Packs/SplunkPy/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,9 @@
+
+#### Classifiers
+##### Splunk - Incoming Mapper
+- Added support for incoming mirroring.
+
+#### Integrations
+##### SplunkPy
```

Comment:
you should note the new parameter names as well

Reply:
any specific format for this?

Reply:
Added support for the *<Param name here>* integration parameter which is used for <info>